playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -17,20 +17,37 @@
       instances: "{{ etcd_names }}"
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
+      g_sub_host_type: "{{ k8s_type }}"
 
   - include: ../../common/openshift-cluster/set_master_launch_facts_tasks.yml
   - include: tasks/launch_instances.yml
     vars:
       instances: "{{ master_names }}"
       cluster: "{{ cluster_id }}"
+      type: "{{ k8s_type }}" # This gets set to master
+      g_sub_host_type: "{{ k8s_type }}" # This gets set to etcd
```

Comment:
how does it get set to 'etcd' when the line above it uses the same variable and gets set to 'master' ?

Either the comment is wrong, or the code is wrong, or I am missing something (and am wrong).   :)